### PR TITLE
Feature/get remote serverinfo

### PR DESCRIPTION
**Pull request by RuneLind**
We now can select hdfs_path, server path and a merge_job can be created.

### CODE REVIEW

File: app/assets/javascripts/hdfs_paths.js.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (vincentp) [inline app/assets/javascripts/hdfs_paths.js.coffee]: I think that you should remove all those empty coffeescript files if you don't need them

File: app/helpers/hdfs_paths_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module HdfsPathsHelper
```
Comment (vincentp) [inline app/helpers/hdfs_paths_helper.rb]: Same for the rails helpers generated by default: remove :)

Comment (RuneLind) [inline app/helpers/hdfs_paths_helper.rb]: I use one of them, but I it's easy to recreate them :)

File: app/assets/javascripts/hdfs_paths.js.coffee (hunk 1 above)
Comment (RuneLind) [inline app/assets/javascripts/hdfs_paths.js.coffee]: sure :)

File: spec/controllers/merge_jobs_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+# is no simpler way to get a handle on the object needed for the example.
+# Message expectations are only used when there is no simpler way to specify
+# that an instance is receiving a specific message.
+
+describe MergeJobsController do
+
+  # This should return the minimal set of attributes required to create a valid
+  # MergeJob. As you add validations to MergeJob, be sure to
+  # update the return value of this method accordingly.
+  def valid_attributes
```
Comment (vincentp) [inline spec/controllers/merge_jobs_controller_spec.rb:26]: It's not really recommended to create method into an Rspec class to set attributes. You should add the valid attributes into a variables, you have 2 options:

let(: valid_attributes) { {} }

or

before do
  @valid_attributes = {}
end

Comment (RuneLind) [inline spec/controllers/merge_jobs_controller_spec.rb:26]: Agree, but this is auto generated by rails scaffold... so I we will need to change this every time we use scaffold then

Comment (vincentp) [inline spec/controllers/merge_jobs_controller_spec.rb:26]: ok, didn't know, pretty weird!

**Pull request merged**